src/components/Content.jsx

```diff
@@ -49,6 +52,9 @@ export const Content = () => {
             />
           }
         />
+        <Route path="/coding" element={<CodingPage />} />
+        <Route path="/football" element={<FootballPage />} />
+        <Route path="/cooking" element={<CookingPage />} />
```

Comment:
You've completed the task of having a different path for each category which is brill 👍 
However, this approach is using a lot of hardcoded repetitive code to implement the different routes and render 3 different components.
I'd like to challenge you to refactor this so that:
- the topics are populated using an API call 
- the paths are populated using these topics
- your three topic components are replaced by one component which takes the topic as a prop